### PR TITLE
Django Tutorial Part 8: User authentication and permissions

### PR DESCRIPTION
**Pull request by VietDung7301**

### CODE REVIEW

**Comment by VietDung7301** [comment]
ready

File: catalog/views.py [hunk 1]
```diff
@@ -60,3 +61,17 @@ class AuthorListView(generic.ListView):
...
+    """Generic class-based view listing books on loan to current user."""
+    model = BookInstance
+    template_name = 'catalog/bookinstance_list_borrowed_user.html'
+    paginate_by = 10
+
+    def get_queryset(self):
+        return (
+            BookInstance.objects.filter(borrower=self.request.user)
+            .filter(status__exact='o')
+            .order_by('due_back')
```
Comment (huongnt-2545) [inline catalog/views.py:76]:
```suggestion
            BookInstance.objects.filter(borrower=self.request.user, status__exact='o')
            .order_by('due_back')
```

**Comment by VietDung7301** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
mợt pull

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**